### ISSUE TITLE
Godot bug: Colour picker doesn't work when window is not default sized

### ISSUE THREAD

**Issue by FuJa0815** [Revolutionary-Games/Thrive#2803]
I think we should remove this color picker: 
![image](https://user-images.githubusercontent.com/30809803/142854627-cfd1bd16-36f6-446b-ba3d-01d5310ba893.png)
It's broken and useless

**Comment by FuJa0815** [comment]
https://godotengine.org/qa/34676/how-do-i-customize-a-colorpicker

**Comment by RacerBG** [comment]
Yes, I can confirm that on low (window) resolutions the colour picker does not work. It even crashed the game but the error log was empty.

**Comment by 84634E1A607A** [comment]
https://github.com/Revolutionary-Games/Thrive/issues/2459#issuecomment-906084578

**Comment by hhyyrylainen** [comment]
Thanks for the link, that's indeed where we were discussing this before.

**Comment by 84634E1A607A** [comment]
Opened an issue:

https://github.com/godotengine/godot/issues/57343

**Comment by 84634E1A607A** [comment]
Let's reopen this to track Godot fix.

**Comment by hhyyrylainen** [comment]
This should be checked that it works in Godot 4.

**Comment by hhyyrylainen** [comment]
I think this is actually a linux specific problem so I'll close this in favour of: https://github.com/Revolutionary-Games/Thrive/issues/5039
